src/styles/main.scss

```diff
@@ -1,3 +1,24 @@
-body {
-  margin: 0;
+@import "utils/normalize.scss";
+@import "utils/variables.scss";
+@import "utils/mixins.scss";
+@import "blocks/page.scss";
+@import "blocks/header.scss";
+@import "blocks/nav-list.scss";
+@import "blocks/catalog.scss";
+@import "blocks/card.scss";
+@import "blocks/stars.scss";
+
+.is-active {
+  color: $button-color;
+
+  &::after {
+    content: "";
+    position: absolute;
+    bottom: 0;
+    right: 0;
+    left: 0;
+    background-color: $button-color;
+    height: 4px;
+    border-radius: 8px;
+  }
 }
```

Comment:
I don't see why you are setting this class here
It should be moved to the corresponding file inside the block

If you are reusing this - move it to mixin